app/about/faq/page.tsx

```diff
@@ -1,7 +1,35 @@
 "use client";
+import {
```

Comment:
> Also cant seem to link myself to the faq page on the preview, not sure why

@HimanshuRanka 

My guess would be that `NEXT_PUBLIC_ABOUT_PAGE_ENABLED` isn't set to `'true'` in the preview environment. Maybe worth taking a look?